pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -469,6 +472,40 @@ def test_to_datetime_tz(self, cache):
         with pytest.raises(ValueError, match=msg):
             pd.to_datetime(arr, cache=cache)
 
+    @pytest.mark.parametrize('cache', [True, False])
+    def test_to_datetime_offset(self, cache):
```

Comment:
can you add a comment what this test does and/or change the name, its not obvious

Reply:
renamed to `test_to_datetime_different_offsets`